src/mgr/Response.cc

```diff
@@ -21,7 +21,7 @@
 Mgr::Response::Response(const Ipc::RequestId aRequestId, const Action::Pointer anAction):
     Ipc::Response(aRequestId), action(anAction)
 {
-    Must(!action || action->name()); // if there is an action, it must be named
+    Must(!action || !action->name().isEmpty()); // if there is an action, it must be named
```

Comment:
Similar to the other change request about removing Must(), this Must() should be dropped IMO. The conversion to SBuf already removes dangers related to nil names (that his Must() is supposed to protect us from AFAICT).


```suggestion
```

Reply:
I disagree on the interpretation. Action names must by non-zero length by their nature as identifiers and as URL paths.

Reply:
> Action names must by non-zero length by their nature as identifiers and as URL paths.

FWIW, my change request does not contradict the above assertion.